core/src/main/java/com/yahoo/imapnio/async/request/OpenFolderActionCommand.java

```diff
@@ -97,6 +122,11 @@ public ByteBuf getCommandLineBytes() throws ImapAsyncClientException {
             byteBuf.writeBytes(sb.toString().getBytes(StandardCharsets.US_ASCII));
         }
 
+        if (this.condStore) {
```

Comment:
See line 112 for computation of unpooled buffer size, condstore size should be of the computation.